sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample04_HookCrudOperations.cs

```diff
@@ -112,13 +112,13 @@ public async Task UpdateHookAsync()
             string originalDescription = hook.Description;
             hook.Description = "This description was generated by a sample.";
 
-            await adminClient.UpdateHookAsync(hookId, hook);
+            await adminClient.UpdateHookAsync(hook);
 
             // Undo the changes to leave the hook unaltered. Skip this step if you intend to keep
```

Comment:
would it be good, that before you make another update, to print in console the new values so you can show the user how things changed?

Reply:
Will update all Update samples in this PR.

Reply:
Updated all tests and samples. Did the same for UpdateDetectionConfig and UpdateAlertConfig as well, since their PRs were already merged.